microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -228,100 +409,47 @@ The updated `manifest.json` that includes the `content-scripts` and `web_accessi
+In injected content scripts, it's common to use jQuery (`$`).  This sample includes a minified version of jQuery, residing in the extension package<!-- todo: check 'package' --> as `lib\jquery.min.js`.
 
-Here's the `content-scripts\content.js` file that gets injected into every browser tab page based on the `content-scripts` section in `manifest.json`:
+Each content script (`popup.js` and `content.js`) runs in an individual sandbox, which means that the jQuery that's injected into the initial popup page isn't shared with the current webpage.
 
-`/content-scripts/content.js` (complete):
 
-```javascript
-chrome.runtime.onMessage.addListener(function(request, sender, sendResponse) {
-    $("body").prepend(
-        `<img  src="${request.url}" id="${request.imageDivId}"
-               class="slide-image" /> `
-    );
-    $("head").prepend(
-        `<style>
-          .slide-image {
-              height: auto;
-              width: 100vw;
-          }
-        </style>`
-    );
-    $(`#${request.imageDivId}`).click(function() {
-        $(`#${request.imageDivId}`).remove(`#${request.imageDivId}`);
-    });
-    sendResponse({ fromcontent: "This message is from content.js" });
-});
-```
-
-All that the above JavaScript does is to register a `listener` using the `chrome.runtime.onMessage.addListener` Extension API method.  This listener waits for messages like the one you sent from the `popup.js` described earlier with the `chrome.tabs.sendMessage` Extension API method.
-
-The first parameter of the `addListener` method is a function whose first parameter, request, is the details of the message being passed in.  Remember, from `popup.js`, when you used the `sendMessage` method, those attributes of the first parameter are `url` and `imageDivId`.
-
-When an event is processed by the listener, the function that is the first parameter is run.  The first parameter of that function is an object that has attributes as assigned by `sendMessage`.  That function simply processes the three jQuery script lines.
-
-*   The first script line appends an `img` element right below the `body` of your browser tab that has the `slide-image` class assigned as well as the `imageDivId` as the ID of that image element.
+<!-- ------------------------------ -->
+#### Understanding the thread
 
-*   The second script line dynamically inserts into the DOM header a **\<style\>** section that you must assign as a `slide-image` class to your `img` element.
+Each tab page (and extension) runs in its own thread.
 
-*   The third script line adds a `click` event that covers the entire image allowing the user to select anywhere on the image and that image is removed from the page (along with it is event listener).
+Even if the browser tab has JavaScript running on it on the loaded web page, any content that's injected doesn't have access to that JavaScript.  The injected JavaScript only has access to the actual DOM that's loaded in that browser tab.
```

Comment:
```suggestion
Even if the browser tab contains a web page that runs JavaScript code, the `content.js` script that's injected in that browser tab by the extension doesn't have access to that JavaScript code.  The injected content script only has access to the DOM of the web page.
```

Reply:
result: the Suggested change, except webpage not web page

Reply:
@captainbrosset - corp style: webpage is one word
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/web-world-wide-web-www - "The following terms that include _web_ are one word: webpage, website,"